src/gui/gui_outliner_window.hpp

```diff
@@ -696,8 +696,21 @@ class outliner_listbox : public listbox2_base<outliner_data> {
 		if(get_filter(state, outliner_filter::gp_influence)) {
 			row_contents.push_back(outliner_filter::gp_influence);
 			auto old_size = row_contents.size();
-			state.world.nation_for_each_gp_relationship_as_great_power(state.local_player_nation,
-					[&](dcon::gp_relationship_id grid) { row_contents.push_back(grid); });
+			state.world.nation_for_each_gp_relationship_as_great_power(state.local_player_nation, [&](dcon::gp_relationship_id grid) {
+				switch(state.world.gp_relationship_get_status(grid) & nations::influence::level_mask) {
+				case nations::influence::level_neutral:
```

Comment:
this is not going to make people happy; people want the nations that they are influencing to show up there, even if they are currently at neutral or less. I also note that fixing whatever was going wrong with influence in gfm should also help